cloudprem/main.tf

```diff
@@ -314,7 +314,6 @@ module "webhooks" {
   rds_pass = random_password.primary_database.result
 
   eks_sg = module.eks_cluster.cluster_primary_security_group_id
-  eks_cluster = module.eks_cluster
```

Comment:
This is a little surprising. Doesnt the frontegg module spray a bunch of helm charts into the cluster? I don't see how it would be able to connect?

Reply:
The helm provider implicitly inherits the eke cluster context from the calling module so no direct connection is required.

Reply:
Ah! So the helm provider picks up the connection and inherits? Nice.